exercises/dnd-character/example.exs

```diff
@@ -0,0 +1,42 @@
+defmodule DndCharacter do
+  @type t :: %__MODULE__{
+    strength: pos_integer(),
+    dexterity: pos_integer(),
+    constitution: pos_integer(),
+    intelligence: pos_integer(),
+    wisdom: pos_integer(),
+    charisma: pos_integer(),
+    hitpoints: pos_integer()
+  }
```

Comment:
I think putting the __MODULE__ is important on my 3rd thought through it to have as an example for others.  This is better.